docs/Source_Sink_Functions.md

```diff
@@ -0,0 +1,12 @@
+# Source and sink functions
+
+Create new source and sink functions in Go. 
+
+## Required guidelines
+
+- Source or sink functions must conform to the following examples: 
+    [sql](https://github.com/influxdata/flux/tree/master/stdlib/sql)
+    [http](https://github.com/Anaisdg/flux/tree/master/stdlib/http)
+- You must submit a unit test in [testing](https://github.com/influxdata/flux/tree/master/stdlib/testing).
+- You must submit an end-to-end test in [testdata](https://github.com/influxdata/flux/tree/master/stdlib/testing/testdata). 
```

Comment:
this isn't really do-able for source/sink since these tests require a server to be active, and community members don't have the authorization to add a running database to our testing platform.